AISKU/src/Initialization.ts

```diff
@@ -50,6 +50,13 @@ export class Initialization implements IApplicationInsights {
         snippet.version = snippet.version || 2.0; // Default to new version
         let config: IConfiguration & IConfig = snippet.config || ({} as any);
 
+        if (config.connectionString) {
+            const cs = ConnectionStringParser.parse(config.connectionString);
+            const ingest = cs.ingestionendpoint;
+            config.endpointUrl = ingest ? `${ingest}/v2/track` : config.endpointUrl; // only add /v2/track when from connectionstring
+            config.instrumentationKey = cs.instrumentationkey || config.instrumentationKey;
+        }
+
         // ensure instrumentationKey is specified
         if (config && !config.instrumentationKey) {
```

Comment:
You can specify connectionString instead of instrumentationKey, here it will override the config set up previously